api/v1alpha1/groupversion_info.go

```diff
@@ -10,7 +7,7 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects.
-	GroupVersion = schema.GroupVersion{Group: "tinkerbell.org", Version: "v1alpha1"}
+	GroupVersion = schema.GroupVersion{Group: "operator.tinkerbell.org", Version: "v1alpha1"}
```

Comment:
I think we can leave this as `tinkerbell.org` creating `stack.tinkerbell.org` as the resource. This is based on the feelers I put out.

Reply:
Done